src/app/collective-rewards/hooks/useGetCycleStartAndDuration.ts

```diff
@@ -0,0 +1,21 @@
+import { EpochTimeKeeperAbi } from '@/lib/abis/v2/EpochTimeKeeperAbi'
+import { SponsorsManagerAddress } from '@/lib/contracts'
+import { useReadContract } from 'wagmi'
+
+//TODO: check if it makes sense to use a hook or the refetch interval
```

Comment:
at the moment the only reason for havign this hook is to have a shared function

Reply:
I'm not sure what I left that to do.